editoast/src/core/simulation.rs

```diff
@@ -623,4 +631,31 @@ mod tests {
             physics_consist.traction_engine.rolling_resistance,
         );
     }
+
+    #[test]
+    fn physics_consist_compute_gamma() {
+        let mut physics_consist = create_physics_consist();
+        physics_consist.traction_engine.const_gamma = 5.0;
```

Comment:
I know it's a test value but 5.0 is extremely high for a gamma, could you set it a 1.0 or something like this ?